modules/troubleshoot/partials/inspect-bundle.adoc

```diff
@@ -0,0 +1,541 @@
+
+After downloading the debug bundle files, you can inspect the contents to debug your cluster. This section provides some useful data points to check while troubleshooting.
+
+Most files in the debug bundle are JSON files. To make it easier to read these files, this section uses jq. To install jq, see the https://stedolan.github.io/jq/download/[jq downloads page^].
+
+=== View the version of Redpanda on all brokers
+
+```bash
+cat admin/brokers.json | jq '.[] | .version'
+```
+
+Example output:
+
+[,json,role=no-copy,subs=attributes+]
+----
+"{latest-redpanda-tag}"
+"{latest-redpanda-tag}"
+"{latest-redpanda-tag}"
+----
+
+=== View the maintenance status of all brokers
+
+```bash
+cat admin/brokers.json | jq '.[] | .node_id, .maintenance_status'
+```
+
+.Example output
+[%collapsible]
+====
+```json
+0
+{
+  "draining": false,
+  "finished": false,
+  "errors": false,
+  "partitions": 0,
+  "eligible": 0,
+  "transferring": 0,
+  "failed": 0
+}
+1
+{
+  "draining": false,
+  "finished": false,
+  "errors": false,
+  "partitions": 0,
+  "eligible": 0,
+  "transferring": 0,
+  "failed": 0
+}
+2
+{
+  "draining": false,
+  "finished": false,
+  "errors": false,
+  "partitions": 0,
+  "eligible": 0,
+  "transferring": 0,
+  "failed": 0
+}
+```
+====
+
+=== View the cluster configuration
+
+```bash
+cat admin/cluster_config.json | jq
+```
+
+.Example output
+[%collapsible]
+====
+```json
+{
+  "abort_index_segment_size": 50000,
+  "abort_timed_out_transactions_interval_ms": 10000,
+  "admin_api_require_auth": false,
+  "aggregate_metrics": false,
+  "alter_topic_cfg_timeout_ms": 5000,
+  "append_chunk_size": 16384,
+  "auto_create_topics_enabled": false,
+  "cloud_storage_access_key": null,
+  "cloud_storage_api_endpoint": null,
+  "cloud_storage_api_endpoint_port": 443,
+  "cloud_storage_azure_container": null,
+  "cloud_storage_azure_shared_key": null,
+  "cloud_storage_azure_storage_account": null,
+  "cloud_storage_bucket": null,
+  ...
+  "target_quota_byte_rate": 2147483648,
+  "tm_sync_timeout_ms": 10000,
+  "topic_fds_per_partition": 5,
+  "topic_memory_per_partition": 1048576,
+  "topic_partitions_per_shard": 1000,
+  "topic_partitions_reserve_shard0": 2,
+  "transaction_coordinator_cleanup_policy": "delete",
+  "transaction_coordinator_delete_retention_ms": 604800000,
+  "transaction_coordinator_log_segment_size": 1073741824,
+  "transactional_id_expiration_ms": 604800000,
+  "tx_log_stats_interval_s": 10,
+  "tx_timeout_delay_ms": 1000,
+  "wait_for_leader_timeout_ms": 5000,
+  "zstd_decompress_workspace_bytes": 8388608
+}
+```
+====
+
+=== Check Enterprise Edition license keys
+
+```bash
+cat admin/license.json | jq
+```
+
+.Example output
+[%collapsible]
+====
+```json
+{
+  "loaded": false,
+  "license": {
+    "format_version": 0,
+    "org": "",
+    "type": "",
+    "expires": 0,
+    "sha256": ""
+  }
+}
+```
+====
+
+=== View metadata about the Redpanda data directory
+
+To check the size of the directories and look for anomalies:
```

Comment:
How about a "for example,..." that points out a couple common anomalies that users should watch out for?

Reply:
It's difficult because it's all a bit subjective and dependant on the dataset. Maybe just if one partition of a given topic is notably larger than others, or contains a lot more data that you would expect for that topic.